bindings/netframework/ElectionGuard.NetFramework/Tests/ElectionGuard.Tests/TestManifest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NUnit.Framework;
+
+using ElectionGuardInterop;
+using ElectionGuard.Tests.Mocks;
+
+namespace ElectionGuard.Tests
+{
+    [TestFixture]
+    class TestManifest
+    {
+        [Test]
+        public void Test_Can_Construct_InternalManifest_From_Manifest()
+        {
+            var manifest = ManifestGenerator.GetMinimalFakeJeffersonCountyManifest();
```

Comment:
we take the manifest and all of it's metadata, and then create an `InternalManifest` which effectively copies only the necessary components, and then also fills in the placeholder selections for the contests.  that is all handled transparently through the constructor